### PR TITLE
add css exclusion class to video-player mutation observer

### PR DESCRIPTION
**Pull request by pitrackster**
allow to exclude a video tag from video-player mutation observer and videojs appliance by adding a css class ('not-video-js')

### CODE REVIEW

**Comment by Elorfin** [comment]
LGTM

**Comment by stefk** [comment]
@pitrackster, you should be able to fix the build issue now that #757 is merged.

File: plugin/video-player/Resources/modules/watcher/observe.js [hunk 1]
```diff
@@ -17,8 +19,10 @@ export default function observe (selector, callback, containers = [document.body
               keepGoing = false
             }
           })
+          // check if the element has the exclude class
+          let exclude = el.className.indexOf(excludeClass) > -1
```
Comment (kitan1982) [inline plugin/video-player/Resources/modules/watcher/observe.js]: Shouldn't you use "const" here instead of "let" ?

**Comment by kitan1982** [comment]
LGTM

**Comment by Elorfin** [comment]
LGTM

**Pull request merged**